Library/Formula/parse-cli.rb

```diff
@@ -0,0 +1,17 @@
+class ParseCli < Formula
+  homepage "https://parse.com"
+  url "https://www.parse.com/downloads/cloud_code/parse",
```

Comment:
I understand the updating issue, 

At this point I and we (parse.com) can only promise that we will be dilligent in updating the formula. 
I agree on the principle of having static links for versions of the cli. We will work towards that.

Sadly, I do not think this can happen right now